tests/cupy_tests/creation_tests/test_from_data.py

```diff
@@ -475,24 +439,77 @@ def test_asfortranarray_cuda_array_zero_dim_dtype(
         return xp.asfortranarray(a, dtype=dtype_b)
 
 
-class DummyObjectWithCudaArrayInterface(object):
+max_cuda_array_interface_version = 2
 
-    def __init__(self, a, mask=None):
+
+@testing.gpu
+@testing.parameterize(*testing.product({
+    'ver': tuple(range(max_cuda_array_interface_version+1)),
+}))
+class TestFromExternalData(unittest.TestCase):
```

Comment:
Let's use "cuda array interface" as the test name.

```suggestion
class TestCudaArrayInterface(unittest.TestCase):
```